unifi-cloudflared-dns-over-https.sh

```diff
@@ -71,59 +71,59 @@ echo ""
 
 # install cloudflared, exit if failure
 if ! apt-get install cloudflared -y ; then
-    echo "error installing cloudflared package, exiting"
-    exit 1
+	echo "error installing cloudflared package, exiting"
+	exit 1
 fi
 
 # get path of cloudflared
 BIN_PATH=$(which cloudflared)
-if [ "" == "$BIN_PATH" ] ; then
-    echo "Cannot find cloudflared executable, exiting"
-    exit 1
+if [ "" = "$BIN_PATH" ] ; then
+	echo "Cannot find cloudflared executable, exiting"
+	exit 1
 fi
 
 # test for presence of unit file
-if [[ -e $UNIT_FILE ]] ; then
-    echo "$UNIT_FILE already exists, skipping setup"
+if [ -e $UNIT_FILE ] ; then
+	echo "$UNIT_FILE already exists, skipping setup"
 else
-    if [ "" == "$CLOUDFLARE_DNS_OVER_HTTPS_URL" ] ; then
-        CLOUDFLARE_DNS_OVER_HTTPS_URL="https://cloudflare-dns.com/dns-query"
-        echo "Using default Cloudflare DoH endpoint."
-    else
-        echo "Using $CLOUDFLARE_DNS_OVER_HTTPS_URL as DoH endpoint."
-    fi
-
-    # write unit file
-    cat << EOF > $UNIT_FILE
-[Unit]
-Description=cloudflared DNS proxy
-After=network.target
-
-[Service]
-TimeoutStartSec=0
-Type=Simple
-ExecStart=$BIN_PATH proxy-dns --address 127.0.0.53 --upstream $CLOUDFLARE_DNS_OVER_HTTPS_URL
-Restart=on-failure
-RestartSec=5s
-
-[Install]
-WantedBy=multi-user.target
+	if [ "" = "$CLOUDFLARE_DNS_OVER_HTTPS_URL" ] ; then
+		CLOUDFLARE_DNS_OVER_HTTPS_URL="https://cloudflare-dns.com/dns-query"
+		echo "Using default Cloudflare DoH endpoint."
+	else
+		echo "Using $CLOUDFLARE_DNS_OVER_HTTPS_URL as DoH endpoint."
+	fi
+
+	# write unit file
+	tee $UNIT_FILE <<- EOF > /dev/null
```

Comment:
I gotta know - what's the benefit here?  Speed?

Reply:
I doubt there is a speed benefit, I just like the target file being part of the command options instead of in the goofy heredoc syntax. This is very much a personal preference, not a real issue.